components/cam/src/physics/crm/crm_physics.F90

```diff
@@ -310,9 +310,9 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
 
    use crm_state_module,       only: crm_state_type
    use crm_rad_module,         only: crm_rad_type, crm_rad_initialize, crm_rad_finalize
-   use crm_input_module,       only: crm_input_type
-   use crm_output_module,      only: crm_output_type, crm_output_initialize, crm_output_finalize
    use crm_ecpp_output_module, only: crm_ecpp_output_type
+   use crm_input_module
```

Comment:
Why remove the `only:` clause?

Reply:
We should definitely keep the only clause in.

Reply:
ok, I see why he did this now, but I'm not sure we want to make this change. We should really have a larger discussion about changes this big.